content/Video_Display.md

```diff
@@ -290,46 +290,46 @@ of this brightness correction.
 
 Writing to this register launches a DMA transfer from ROM or RAM to OAM
 memory (sprite attribute table). The written value specifies the
-transfer source address divided by 100h, that is, source & destination are:
+transfer source address divided by $100, that is, source and destination are:
 
 ```
-Source:      XX00-XX9F   ;XX in range from 00-F1h
-Destination: FE00-FE9F
+Source:      $XX00-$XX9F   ;XX = $00 to $F1
```

Comment:
Wait... why up to $F1? It shouldn't go higher than $DF... and strictly speaking, all values do *something*, it's just "weird" when it gets higher than $FE, and $E0+ access echo RAM...

Reply:
Fixed